package.json

```diff
@@ -105,7 +101,11 @@
   "dependencies": {
     "bootstrap": "^3.3.6",
     "font-awesome": "^4.6.3",
-    "normalize.less": "^1.0.0"
+    "normalize.less": "^1.0.0",
+    "less": "2.7.2",
+    "autoprefixer": "^7.1.2",
```

Comment:
Please alphabetize dependencies, or run `npm install --save less` to get npm to automatically sort.

---

Also regenerate `package-lock.json`.
npm v5+ pulls dependencies purely based off the lockfile, if the lock is out of sync with the package, the changes here would not appear.